dsn.go

```diff
@@ -51,6 +52,7 @@ type Config struct {
 	MultiStatements         bool // Allow multiple statements in one query
 	ParseTime               bool // Parse time values to time.Time
 	Strict                  bool // Return warnings as errors
+	MaxAllowedPacket	int  // Max packet size allowed
```

Comment:
Please move it after `Loc *time.Location // Location for time.Time values`.
We just put all the bool vars at the end for more efficient struct packing.